BaseFilter/sections/antiadblock.txt

```diff
@@ -1401,16 +1401,22 @@ strtapeadblocker.art#@#.skyscraper.ad
 hotdebrid.com#%#//scriptlet('prevent-element-src-loading', 'script', 'ads.rubiconproject.com/ad/')
 !+ PLATFORM(ios, ext_android_cb)
 @@||ads.rubiconproject.com/ad/*.js$domain=hotdebrid.com
-! https://github.com/AdguardTeam/AdguardFilters/issues/123316
+! https://github.com/AdguardTeam/AdguardFilters/issues/133157
+work.ink#?#.content .col-md .row:has(div[id^="google_ads_"])
```

Comment:
But this is an ad(ad leftover)

Reply:
Yeap. I added the rule to block 2 leftovers on the website.